Chore: update onboarding guide

MarceloRobert opened the pull request:
## Changes
- Added monitor_submissions docs
- Added .keep files to commit required folders and updated .gitignore
- Updated backend Readme.md
- Updated onboarding.md

Closes #1471

JenySadadia reviewed (commented):
Hi, thanks for this Marcelo !
I think this guide is missing the part where we set up local sqlite DB user and grant all permissions to run django migration. 
There should be a script for that.

MarceloRobert commented:
Thanks for reminding me about the SQLite @JenySadadia ! I don't know if it is possible to run the migrations automatically when running locally, so I added an instruction for it on the Readme.md (and also updated it on other places)

JenySadadia commented:
> Thanks for reminding me about the SQLite @JenySadadia ! I don't know if it is possible to run the migrations automatically when running locally, so I added an instruction for it on the Readme.md (and also updated it on other places)

Thanks for the update. I still don't see any mention of creating `dev` user in local sqlite DB. To be able to run the migrations I needed to create the user and grant all permissions for accessing `dashboard` DB.

LucasSantos27 commented on backend/README.md:
```diff
@@ -29,66 +29,96 @@ To find the exact path to the Python executable inside the virtual environment,
+DASH_DB_NAME=dashboard
+DASH_DB_USER=admin
+DASH_DB_PASSWORD=admin
+DASH_DB_HOST=dashboard_db
```
It wouldn't be `localhost` or `127.0.0.1`?

MarceloRobert replied:
True, I just moved it here and forgot to change it. Done.

LucasSantos27 commented on backend/README.md:
````diff
@@ -29,66 +29,96 @@ To find the exact path to the Python executable inside the virtual environment,
+```
 
-## Running server
+If you have both connections and you want to use the local one as the default, the env var `USE_DASHBOARD_DB` can be exported as `True` for a quick flag change.
````
It would be nice to write about some scripts:
- `update_db_7_days.sh`: if we have access to the kcidb, we can sync them
- `migrate-app-db.sh`: run migrations

Besides that, it's important to warn Django that it generates the new migrations
```sh
run python3 manage.py makemigrations <project> --name <migration_name>
```

LucasSantos27 commented on docs/Onboarding.md:
```diff
@@ -28,9 +30,20 @@ This frontend is written in TypeScript and uses the React library.
-1. Check for the Cloud SQL proxy access in your e-mail, follow the instructions there to run proxy.
-2. Check for database access in your e-mail, follow the instructions, try to connect to the kcidb database via CLI and see if it is working.
-3. Spin up the local dashboard-db by starting its docker container and running the [migration script](../backend/migrate-app-db.sh)
+In order to access the production database, you must be granted access to it first - proxy connection and database user. If you don't have access to the production database, you can point the environment variables to a local database, following [these instructions]().
```
These instructions what?

LucasSantos27 commented on docs/Onboarding.md:
```diff
@@ -28,9 +30,20 @@ This frontend is written in TypeScript and uses the React library.
+* If you'll connect the database on Google Cloud, set up your default credentials by following Google's guide [here](https://cloud.google.com/docs/authentication/set-up-adc-local-dev-environment#google-account). The Google Cloud database is deprecated and will be deactivated soon.
+
+* If you'll connect to kcidb on Azure, create a new SSH key and add it to your ssh agent. You can follow this [Github guide](https://docs.github.com/en/authentication/connecting-to-github-with-ssh/generating-a-new-ssh-key-and-adding-it-to-the-ssh-agent) for it (but you don't need to add the key to your Github account, it can stay only in your PC).
+* Share the public SSH key to the database maintainer, so that he/she can give you access to the SSH tunnel.
```
nit: singular they

ref https://en.wikipedia.org/wiki/Singular_they
